econml/iv/dml/_dml.py

```diff
@@ -526,7 +526,7 @@ def score(self, Y, T, Z, X=None, W=None, sample_weight=None):
             The MSE of the final CATE model on the new data.
         """
         # Replacing score from _OrthoLearner, to enforce Z to be required and improve the docstring
-        return super().score(Y, T, X=X, W=W, Z=Z, sample_weight=sample_weight)
+        return super().score(Y, T, X=X, W=W, Z=Z, sample_weight=sample_weight, groups=None)
```

Comment:
should it be groups=groups here?

Reply:
Yes, good catch.  (It doesn't affect the results since groups are never used in scoring, but I'll fix it in the next set of changes).

Reply:
Upon further consideration, I've removed `groups` from the DMLIV and DRIV scoring methods, because they are never used and so there's no point in including them.

The `groups` argument needs to exist on the nuisance models, because the signatures for fit, predict, and score all need to be compatible for how we do cross-fitting, but there's no need for them to pollute the estimators themselves, and indeed our existing classes like LinearDML do not have groups on their scoring methods.